central/hash/manager/deduper_test.go

```diff
@@ -170,11 +170,12 @@ func TestDeduper(t *testing.T) {
 	for _, c := range cases {
```

Comment:
WDYT about adding some test cases for the events that should not be deduped? E.g.: given two identical runtime alert events, both will be processed.